staging/src/k8s.io/client-go/tools/pager/pager.go

```diff
@@ -121,7 +121,6 @@ func (p *ListPager) List(ctx context.Context, options metav1.ListOptions) (runti
 		if list == nil {
 			list = &metainternalversion.List{Items: make([]runtime.Object, 0, options.Limit+1)}
 			list.ResourceVersion = m.GetResourceVersion()
-			list.SelfLink = m.GetSelfLink()
```

Comment:
same comment, this is client-side, so it should continue copying to faithfully propagate the server response

Reply:
done